force-app/main/default/classes/ServiceSessionStatusPicklist.cls

```diff
@@ -0,0 +1,30 @@
+public class ServiceSessionStatusPicklist extends VisualEditor.DynamicPickList {
```

Comment:
There's some documentation that suggests this class may need to be `global1` access to serve the flexipage design property. However there's some likely typos in those docs that make them hard to interpret, and experiments in a 2gp org seem to work fine with simple public scope, even after the flexipage has been cloned.